tensorflow/contrib/learn/python/learn/datasets/__init__.py

```diff
@@ -64,3 +73,34 @@ def load_dataset(name, size='small', test_with_fake_data=False):
     return DATASETS[name](size, test_with_fake_data)
   else:
     return DATASETS[name]()
+
```

Comment:
what's `nit` - Unit of entropy? Sorry, was trying to find it, but the only thing comes to mind is "not important thing"?